src/Makefile

```diff
@@ -33,3 +47,7 @@ pvAccess_SYS_LIBS_WIN32 += ws2_32
 
 include $(TOP)/configure/RULES
 
+# Can't use EXPAND as generated headers must appear
+# in O.Common, but EXPAND emits rules for O.$(T_A)
+../O.Common/pvaVersionNum.h: pvaVersionNum.h@
+	$(EXPAND_TOOL) $(EXPANDFLAGS) $($@_EXPANDFLAGS) $< $@
```

Comment:
@anjohnson Can you suggest some way to avoid a custom rule here?  I couldn't see an obvious way to get $(EXPAND) to install to O.Common.